components/src/Select/Select.story.js

```diff
@@ -67,10 +77,10 @@ storiesOf("Select", module)
       />
     </>
   ))
-  .add("Select as a controlled component", () => (
+  .add("as a controlled component", () => (
     <SelectWithState placeholder="Please select inventory status" options={options} labelText="Inventory status" />
   ))
-  .add("Set to disabled", () => (
+  .add("set to disabled", () => (
     <Select placeholder="Please select inventory status" options={options} disabled labelText="Inventory status" />
   ))
   .add("with error message", () => (
```

Comment:
The error message box, when open, has a red border with blue drop-shadow. Is this intended?

Reply:
Yes, as both those states are true (it is errored: red border, and focused: blue shadow). Our old Select was missing focus states so this wasn't apparent. If there is confusion we can make changes but I like starting with the simplicity of just showing both true states as-is.